TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleOp/Mechanisms/TransferController.java

```diff
@@ -62,5 +77,56 @@ private boolean delay() {
         double finalTS = bufferTime.milliseconds();
         return (finalTS - startTS) >= DELAY_MS;
     }
-}
 
+    public void pos_up(){
+        int tempPos = slideIndexPos + index_increment;
+        if(tempPos < MAX_SLIDE_HEIGHT){
+            slideIndexPos = tempPos;
+            slides.setTargetPos(slides.mmToTicks(slideIndexPos));
+        }
+    }
+
+    public void pos_down(){
+        int tempPos = slideIndexPos - index_increment;
+        if(tempPos >= min_slide_height){
+            slideIndexPos = tempPos;
+            slides.setTargetPos(slides.mmToTicks(slideIndexPos));
+        }
+    }
+
+    public void offset_up(){
+        /*
+        slideIndexPos += OFFSET_INCREMENT;
+        min_slide_height += OFFSET_INCREMENT;
+        slides.setTargetPos(slides.mmToTicks(slideIndexPos));
+         */
+        int tempMinPos = min_slide_height + OFFSET_INCREMENT;
+        int tempIndexPos = slideIndexPos + OFFSET_INCREMENT;
+        if(tempMinPos < MAX_SLIDE_HEIGHT && tempIndexPos < MAX_SLIDE_HEIGHT){
+            min_slide_height = tempMinPos;
+            slideIndexPos = tempIndexPos;
+            slides.setTargetPos(slides.mmToTicks(slideIndexPos));
+        }
+    }
+    public void offset_down(){
+        /*
```

Comment:
what is this commented out code for? if it's for the test bench only make a comment, if it's not used it can be removed.